### ISSUE TITLE
Managing multiple versions of Elixir?

### ISSUE THREAD

**Issue by cclark** [ElixirTalk/elixirtalk#21]
As someone coming from the Scala/JVM world but who has had to dabble in Ruby and Python oftentimes to get my dev toolchain working I've grown to love tools like rbenv and pyenv to insulate me from destroying a system install. But it is also really a nice way to be able to try out a newer version. Elixir isn't moving at a fast pace but I was recently working through the Pragmatic Studio video series on Elixir and found a few things that differed. I noticed I was using 1.5 and the videos were shot on 1.4. The ChangeLog didn't highlight anything specific but I thought immediately that I could easily install the Elixir equivalent of these version managers (and crossed my fingers that it wasn't based on RVM). Heck if [jenv](http://www.jenv.be) exists for the Java world I figured there had to be something...

I came upon [Kiex](https://github.com/taylor/kiex) which I was suspicious of for not following the <Language>env convention but was pleasantly surprised to see the Alternatives section included. I also came across a thread called [Elixir Version Managers in the Elixir Forums](https://elixirforum.com/t/elixir-version-managers/317) with the last update from more than 1.5 years ago.

As someone new to the Elixir ecosystem I have a few questions:
* Is this a solved a problem and everyone uses one of Kiex, ExENV or ASDF but this is tribal knowledge that isn't prevalent?
* Is there something about Erlang and/or Elixir that makes this a non-issue?

While this question is very specific, a show with an overview of the tools you've settled on for Elixir development could be very interesting and might help eliminate some of the gyrations someone new to Elixir must go through to get up to speed quickly due to various yak shaves of trying out different tools.

**Comment by desmondmonster** [comment]
https://soundcloud.com/elixirtalk/episode-108-managing-multiple-elixir-versions-how-much-erlang-should-you-know-and-thinking-functionally

**Comment by harlantwood** [comment]
TL;DR: use [ASDF](https://github.com/asdf-vm/asdf).  

For more insight, the episode has lots of details and reasons why.
